src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -235,7 +236,7 @@ class InstrumentCodeAction implements Action<Task> {
             }
 
             ant.javac(source: getSourceCompatibility(), target: getTargetCompatibility(), encoding: getEncoding(),
-                      executable: getExecutable())
+                      executable: getExecutable(), debug: getDebug())
```

Comment:
I found that groovyc does not have a debug equivalent flag. I suppose groovy always compiles with symbols since it does so much with reflection? At any rate I was able to make a very effective integration test for this. The answer was super simple after all. Create a unit test in the integration project that catches a thrown exception from the code under test and look for the StackTraceElement line number. When using debug that is != -1 and -1 when debug is off. Really easy after all. I am rebasing and merging this with the added integration test.